subgraphs/_reference_/src/sdk/protocols/generic/config.ts

```diff
@@ -0,0 +1,50 @@
+import { Versions } from "../../../../../../deployment/context/interface";
+
+import { ProtocolConfigurer } from "../config";
+import { BridgePermissionType } from "./enums";
+
+export interface Config extends ProtocolConfigurer {
+  getPermissionType(): BridgePermissionType;
+}
+
+export class Config implements Config {
```

Comment:
Since the generic schema doesn’t require any custom config parameters you can reuse the class on `/sdk/protocols/config.ts`

Reply:
Yup - Updated to use the base config class.